builtin/mainmenu/dlg_clients_list.lua

```diff
@@ -0,0 +1,49 @@
+-- Luanti
+-- Copyright (C) 2024 siliconsniffer
+-- SPDX-License-Identifier: LGPL-2.1-or-later
+
+
+local function clients_list_formspec(dialogdata)
+	local TOUCH_GUI = core.settings:get_bool("touch_gui")
+	local clients_list = dialogdata.server.clients_list
+	local servername   = dialogdata.server.name
+
+	local function fmt_formspec_list(clients_list)
+		local escaped = {}
+		for i, str in ipairs(clients_list) do
+			escaped[i] = core.formspec_escape(str)
+		end
+		return table.concat(escaped, ",")
+	end
+
+	local formspec = {
+		"formspec_version[8]",
+		"size[6,9.5]",
+		TOUCH_GUI and "padding[0.01,0.01]" or "",
+		"hypertext[0,0;6,1.5;;<global margin=5 halign=center valign=middle>",
+			fgettext("This is the list of clients connected to\n$1",
```

Comment:
This string sounds awkward, should be something like "List of clients connected to $1:"

Reply:
https://github.com/luanti-org/luanti/pull/15736#pullrequestreview-2604283778